### ISSUE TITLE
My antivirus and Chrome said it was a trojan (and I didn't listen)

### ISSUE THREAD

**Issue by TheLizardMann** [solsticegamestudios/GModCEFCodecFix#128]
So I go to download the windows executable and both my antivirus and Chrome stopped it last minute saying that it was "Dangerous."

I (being a dumbass) decided to press on redownloading, forcing Chrome to finish downloading, and making my antivirus de-quarantine the file. When I realized my error, I had two partly downloaded file that refused to be deleted, forcing me to put windows into safe mode just to delete them.

Then, when I was booting my PC out of safe mode, I receive a BLUE SCREEN! And then provide to spend the next few hours finding some way to fix my PC, after nearly breaking down ofc.

TL;DR, I think the files are infected

**Comment by WinterPhoenix** [comment]
They are definitely not infected, and we definitely didn't cause whatever problem is causing you to blue screen. Please see #126.
